static/src/javascripts/projects/commercial/modules/messenger/resize.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import { register } from 'commercial/modules/messenger';
+
+type Specs = {
+    width?: string,
+    height?: string,
+};
+
+const normalise = (length: string | number): string => {
+    const lengthRegexp = /^(\d+)(%|px|em|ex|ch|rem|vh|vw|vmin|vmax)?/;
+    const defaultUnit = 'px';
+    const matches = String(length).match(lengthRegexp);
+    if (!matches) {
+        return '';
+    }
+    return matches[1] + (matches[2] === undefined ? defaultUnit : matches[2]);
+};
+
+const resize = (
+    specs: Specs,
+    iframe: HTMLElement,
+    adSlot: HTMLElement
+): ?Promise<any> => {
+    if (
+        !specs ||
+        !('height' in specs || 'width' in specs) ||
+        !iframe ||
+        !adSlot
+    ) {
+        return null;
```

Comment:
Null is quite nice for testing against 😜 

I'm happy to change the return, but am not certain what the purpose of the null is (or the side effects) So I would like to split the changes into smaller PRs.

This will hopefully make regressions easier to catch... Although I'm still kinda wary of changing the module and it's tests at the same time. Ah well, at least I can (personally) guarantee that all the legacy tests passed right up until I deleted them!